proto/qdrant/cloud/iam/v1/iam.proto

```diff
@@ -307,6 +379,32 @@ enum UserStatus {
   USER_STATUS_DELETED = 3;
 }
 
+// LegalDocumentType specifies the type of a legal document.
+enum LegalDocumentType {
+  // Unspecified document type.
+  LEGAL_DOCUMENT_TYPE_UNSPECIFIED = 0;
+  // Terms of Service document.
+  LEGAL_DOCUMENT_TYPE_TERMS_OF_SERVICE = 1;
+  // Privacy Policy document.
+  LEGAL_DOCUMENT_TYPE_PRIVACY_POLICY = 2;
+  // Service Level Agreement document.
+  LEGAL_DOCUMENT_TYPE_SLA = 3;
+}
+
+// UserConsentAction specifies the action taken by a user regarding a consent.
+enum UserConsentAction {
```

Comment:
not a big deal, but if we change change this from `Action` to `Status`, the values could be consistent with the PENDING one as well.

This could be:

`USER_CONSENT_STATUS_ACCEPTED`
`USER_CONSENT_STATUS_REVOKED`
`USER_CONSENT_STATUS_PENDING`

This way we could also use the "standard" attribute `last_modified_at` instead of `action_at` or `last_actioned_at`.